linker.go

```diff
@@ -0,0 +1,134 @@
+package tests
+
+import (
+	"github.com/beyondstorage/go-storage/v4/types"
+	"github.com/google/uuid"
+	. "github.com/smartystreets/goconvey/convey"
+	"testing"
+)
+
+func TestLinker(t *testing.T, store types.Storager) {
+	Convey("Given a basic Storager", t, func() {
+		l, ok := store.(types.Linker)
+		So(ok, ShouldBeTrue)
+
+		Convey("When create a link object", func() {
+			path := uuid.New().String()
+			_ = store.Create(path)
```

Comment:
`Create` is a local function without sending API/RPC call，so `path` is a non-existent resource for server here. Maybe we can call `Write` to upload an object or file here as the source?

Reply:
Got it!